### PR TITLE
misc changes from sql code review

### PR DESCRIPTION
**Pull request by snopoke**
@NoahCarnahan 
cc @dimagi/scale-team 

best read commit/commit :fish:

### CODE REVIEW

File: corehq/sql_accessors/sql_templates/case_modified_since.sql [hunk 1]
```diff
@@ -5,7 +5,7 @@ BEGIN
     SELECT NOT exists(
         SELECT 1 FROM form_processor_commcarecasesql
         WHERE form_processor_commcarecasesql.case_id = p_case_id
-          AND form_processor_commcarecasesql.server_modified_on = p_server_modified_on AT TIME ZONE 'UTC'
+          AND form_processor_commcarecasesql.server_modified_on = p_server_modified_on
```
Comment (snopoke) [inline corehq/sql_accessors/sql_templates/case_modified_since.sql]: hmm I didn't mean to commit this - but also don't think it's needed

File: corehq/form_processor/backends/sql/dbaccessors.py [hunk 2]
```diff
@@ -354,7 +354,10 @@ def case_modified_since(case_id, server_modified_on):
     @staticmethod
     def get_case_xform_ids(case_id):
         with get_cursor(CommCareCaseSQL) as cursor:
-            cursor.execute('SELECT form_id FROM get_case_form_ids(%s)', [case_id])
+            cursor.execute(
+                'SELECT form_id FROM get_case_transactions_by_type(%s)',
```
Comment (snopoke) [inline corehq/form_processor/backends/sql/dbaccessors.py]: missing param added later

File: corehq/form_processor/backends/couch/dbaccessors.py [hunk 3]
```diff
@@ -59,7 +59,7 @@ def get_forms(form_ids, ordered=False):
 
     @staticmethod
     def get_form_ids_in_domain_by_type(domain, type_):
-        pass
+        return get_form_ids_by_type(domain, type_)
```
Comment (snopoke) [inline corehq/form_processor/backends/couch/dbaccessors.py]: currently this is only used for SQL domains hence why it hasn't been an issue but thought I'd implement it anyway

File: corehq/tests/nose.py [hunk 4]
```diff
@@ -238,9 +240,9 @@ def get_test_labels(cls, tests):
 
         This should be called after `loadTestsFromNames` has been called.
         """
-        test_apps = set(app for app in settings.INSTALLED_APPS
-                        if app not in settings.APPS_TO_EXCLUDE_FROM_TESTS
-                        and not app.startswith('django.'))
+        test_apps = set(app.name for app in apps.get_app_configs()
+                        if app.name not in settings.APPS_TO_EXCLUDE_FROM_TESTS
+                        and not app.name.startswith('django.'))
```
Comment (snopoke) [inline corehq/tests/nose.py]: pulled out into https://github.com/dimagi/commcare-hq/pull/11744

**Comment by NoahCarnahan** [comment]
All looks good to me!

File: corehq/sql_accessors/migrations/0027_audit_fixups.py [hunk 5]
```diff
@@ -19,7 +19,7 @@ class Migration(migrations.Migration):
 
     operations = [
         HqRunSQL(
-            "DROP FUNCTION IF EXISTS get_all_reverse_indices(TEST[])",
```
Comment (benrudolph) [inline corehq/sql_accessors/migrations/0027_audit_fixups.py]: haha surprised this even ran

Comment (snopoke) [inline corehq/sql_accessors/migrations/0027_audit_fixups.py]: Same here.

**Pull request merged**